Update quickstart guide (#980)
rsksmart/devportal: content/rsk-devportal/guides/quickstart/hardhat/interact-with-frontend.md

## Project Setup

> 1. Create a new folder called `frontend` and navigate to the directory:

Reviewer: On cloning the master branch, there's already a frontend folder. Add a note to those using the quick start repo, to only cd into the frontend directory

## Project Setup

1. Create a new folder called `frontend` and navigate to the directory:
    
```shell
  mkdir frontend
  cd frontend
```

> > However, if you clone the master branch, there's already a frontend folder, just `cd` into the frontend directory

Reviewer: ```suggestion
> Note: If you use the quick start repo on `master`, there's already a frontend folder. You can `cd` into the frontend directory.
```

  cd frontend
```

> 2. Move the file from the `compiled contract` to the frontend directory

Author: ```suggestion
2. Move the file from the `compiled contract` to the frontend directory.
```

> 3. In the frontend directory, initialize a Node.js Project

Author: ```suggestion
3. In the frontend directory, initialize a Node.js Project:
```

```shell
  npm init -y
```

> 4. Install Ethers.js

Author: ```suggestion
4. Install Ethers.js:
```
